localdev/config/binoculars/base.yaml

```diff
@@ -0,0 +1 @@
+corsAllowedOrigins: ["http://localhost:3000", "http://localhost:8089"]
```

Comment:
Is this going to work as expected being localhost?

Reply:
maybe? this will be the address used in your browser, likely localhost

Reply:
We maybe want to support configuring this with ENV vars or something similar?